web/src/main/java/org/cbioportal/proxy/ProxyController.java

```diff
@@ -44,14 +46,100 @@ public String proxy(@RequestBody(required = false) String body, HttpMethod metho
         ).getBody();
     }
 
+    // TODO remove this endpoint after external projects switch to the new proxy
     @RequestMapping("/oncokb/**")
-    public String proxyOncokb(@RequestBody(required = false) String body, HttpMethod method, HttpServletRequest request)
-        throws URISyntaxException {
+    public String proxyOncokb(
+        @RequestBody(required = false) String body,
+        HttpMethod method,
+        HttpServletRequest request
+    ) throws URISyntaxException {
+        return exchangeOncokbData(
+            body,
+            request.getPathInfo().replaceFirst("/oncokb", ""),
+            request.getQueryString(),
+            method,
+            getOncokbHeaders(request)
+        );
+    }
+    
+    @RequestMapping("/monkeykb/**")
```

Comment:
Tentative name for the new endpoint. We can make this one non human readable as well.